autosubmit/autosubmit.py

```diff
@@ -155,7 +155,7 @@ def experiment_data(self):
         script_dir = os.path.join(script_dir, os.path.pardir)
 
     version_path = os.path.join(script_dir, 'VERSION')
-    readme_path = os.path.join(script_dir, 'README')
+    readme_path = os.path.join(script_dir, 'README.md')
```

Comment:
Picked the `readme` command to test, and it was broken :disappointed_relieved: . This fixes it.